README.md

```diff
@@ -215,6 +215,10 @@ masters:
  - name:  lava-master	name of the master
     host: name		name of the host running lava-master (default to "local")
     webadmin_https:	Does the LAVA webadmin is accessed via https
+    zmq_auth: True/false	Does the master requires ZMQ authentication.
```

Comment:
s/false/False/  ?   

I can't remember if the YAML parser is case-sensitive, but the doc should be consistent (e.g. the persistent_db entry just below uses upper case)

Reply:
done